src/extension.ts

```diff
@@ -27,11 +27,12 @@ export async function activate(context: vscode.ExtensionContext, isRestart = fal
         verifyExperimentalAffinity();
     }
 
-    const outputChannel = vscode.window.createOutputChannel(EXT_NAME, { log: true });
-    disposables.push(outputChannel);
+    const logOutputChannel = vscode.window.createOutputChannel(`${EXT_NAME} (logs)`, { log: true });
+    const userMessageOutputChannel = vscode.window.createOutputChannel(`${EXT_NAME} (messages)`);
```

Comment:
Personal hobby: Can it be changed to `messageOutputChannel`?

Reply:
I don't feel strongly. If you do, I'll change it. My thought was it was a bit clearer to say "user" because all output channels display messages...

Reply:
Nvim messages aren't necessarily user-initiated; they could come from plugin printouts. Both `log` and `message` clearly indicate their respective purposes.

Reply:
Eh on second thought, I think yours is better